style.css

```diff
@@ -73,10 +73,6 @@ p {
 
 /* Logo */
 .logo {
```

Comment:
@jmalvarez-narra the default font color of the logo and navigation is dark. in order to use the white color version, we're adding `.header-light` class to `header` element

Reply:
Homepage only uses the dark color version, while all other pages uses the white color version.
IF there are no feature image in the page, that's the time we'll use the dark color version of header